schema/json-schema-credential-schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://www.w3.org/2022/credentials/v2/json-schema-credential-schema.json",
+  "description": "JSON Schema for a Verifiable Credential of type JsonSchemaCredential according to the Verifiable Credentials Data Model v2",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "array",
+      "const": [
+        "VerifiableCredential",
+        "JsonSchemaCredential"
+      ]
+    },
+    "credentialSubject": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "const": "JsonSchema"
+        },
+        "jsonSchema": {
```

Comment:
@msporny you can see here the base schema just enables the use of the schema versions the spec allows